packages/@aws-cdk/aws-rds/lib/serverless-cluster.ts

```diff
@@ -389,25 +393,40 @@ export class ServerlessCluster extends ServerlessClusterBase {
 
     this.vpc = props.vpc;
     this.vpcSubnets = props.vpcSubnets;
+    this.subnetGroup = props.subnetGroup;
+    let securityGroups: ec2.ISecurityGroup[] | undefined = props.securityGroups;
```

Comment:
Can you move this local variable closer to where it's first used (above `if (this.vpc !== undefined) {`)?

Reply:
👍🏻